src/SFA.DAS.ApplyService.Application/Apply/Financial/ClarificationFileUploadHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.ApplyService.Domain.Entities;
+
+namespace SFA.DAS.ApplyService.Application.Apply.Financial
+{
+    public class ClarificationFileUploadHandler : IRequestHandler<ClarificationFileUploadRequest, bool>
```

Comment:
Is it possible to prefix this with Add so it's `AddClarificationFileUploadHandler`  as the next story you'll be doing will be removing files

Reply:
yes